Lib/test/test_sys.py

```diff
@@ -803,11 +803,17 @@ def test_sys_flags(self):
                  "dont_write_bytecode", "no_user_site", "no_site",
                  "ignore_environment", "verbose", "bytes_warning", "quiet",
                  "hash_randomization", "isolated", "dev_mode", "utf8_mode",
-                 "warn_default_encoding", "safe_path", "int_max_str_digits")
+                 "warn_default_encoding", "safe_path", "int_max_str_digits",
+                 "gil")
+        attr_types = {
+            "dev_mode": bool,
+            "safe_path": bool,
+            "gil": (int, type(None)),
```

Comment:
The None values means that it's the free-threaded build (or is it for something else?)